github/enterprise_manage_ghes.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+)
+
+// NodeQueryOptions specifies the optional parameters to the EnterpriseService
+// Node management APIS.
+type NodeQueryOptions struct {
+	// UUID filters issues based on the node UUID.
+	UUID string `url:"uuid,omitempty"`
+
+	// ClusterRoles filters The cluster roles from the cluster configuration file.
+	ClusterRoles string `url:"cluster_roles,omitempty"`
+}
+
+// ClusterStatus represents a response from the GetClusterStatus and  GetReplicationStatus method.
```

Comment:
Please make sure that all godoc-style comments are well-formed with punctuation and no double-spaces like this one:

```suggestion
// ClusterStatus represents a response from the GetClusterStatus and GetReplicationStatus method.
```

Reply:
Not that I'm aware, sorry.